src/main/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/OrganisationService.java

```diff
@@ -121,15 +116,16 @@ public void findOrganisationChangesAndInsertIntoOrganisationRefreshQueue() {
             String pageFailMessage = (page == 0 ? "" : ", failed at page " + page);
             processMonitorDto.markAsFailed(exception.getMessage() + pageFailMessage);
             processEventTracker.trackEventCompleted(processMonitorDto);
-            throw exception;
+            return processMonitorDto;
```

Comment:
I think we need to revert the changes in exception handling in this class: as we are only testing happy path scenarios with FTAs.  So error states can stil throw an exception provided the processEventTracker is updated with the failure and completed states.

Reply:
Resolved in f51ecf4dba08db15b851d32c11b0a3ab0a5d909c.